fluentd/configs.d/mux-post-input-filter-tag.conf

```diff
@@ -15,6 +15,8 @@
 <match **>
   @type rewrite_tag_filter
   @label @INGRESS
+  buffer_queue_limit "#{ENV['BUFFER_QUEUE_LIMIT']}"
+  buffer_chunk_limit "#{ENV['BUFFER_SIZE_LIMIT']}"
```

Comment:
@richm, let me double check...  "these" means the buffer_.*_limit in fluentd/configs.d/mux-post-input-filter-tag.conf?  
Any other ones?
Sorry & Thanks!